ods_ci/tests/Resources/Page/Operators/OpenShiftPipelines.resource

```diff
@@ -46,8 +43,10 @@ Install Red Hat OpenShift Pipelines
         ELSE
             ${openshift_pipelines_version}=    Set Variable    pipelines-1.11
```

Comment:
OpenShift 4.14 is not available yet, so OpenShift Pipelines 1.12 won't be tested so much

One option could be test 1.12 also in OpenStack:
`- OpenShift >= 4.12 (OpenStack):       OpenShift Pipelines 1.12`

Reply:
done